web/templates/invite_user_modal.hbs

```diff
@@ -32,6 +32,13 @@
             <textarea rows="2" id="invitee_emails" name="invitee_emails" class="invitee_emails" placeholder="{{t 'One or more email addresses...' }}"></textarea>
         </div>
     </div>
+    <div class="input-group new-style" id="accept-direct-message">
```

Comment:
A better ID would be `receive-direct-message`.

Reply:
I guess you named it such to relate it with the message sent when invite is acccepted but maybe we can still name it something better like `invite-accept-notification`, but we can think of something better.